convert2rhel/pkghandler.py

```diff
@@ -196,7 +195,6 @@ def get_installed_pkg_objects(name=""):
     """Return list with installed package objects. The packages can be
     optionally filtered by name.
     """
-    yum_base = yum.YumBase()
```

Comment:
Is there any problem with the yum module? Without this line, the `yum_base` will be uninitialized.

Reply:
Returned it back as it was